spec/features/user_cannot_access_admin_privileges_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+
+RSpec.describe 'Access', type: :feature do
+  create_user
+    feature 'user' do
+      it 'cannot access admin screens' do
+
+      visit root_path
+      click_link "Login"
+
+      fill_in "Username", with: user_one.username
+      fill_in "Password", with: 'password'
+
+      click_button "Login"
+
+      visit new_admin_category_path
+
+      expect(page).to have_content("404 Page Not Found")
+    end
+
+    it 'cannot visit another users rental history' do
+      user_three = User.create(username: "bob",
+                               password: "whatsup")
+      visit root_path
+      click_link "Login"
+
+      fill_in "Username", with: user_three.username
+      fill_in "Password", with: 'whatsup'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.